finger_exercises/ecmascript/01 - spread/utils.js

```diff
@@ -2,8 +2,6 @@
 // export default
 // Would you like to fix this issue ?
 
-export function isArray(aPossibleArray) {
-  return !aPossibleArray ? aPossibleArray : aPossibleArray.constructor === Array;
-}
+// eslint-disable-next-line import/prefer-default-export
+export const isArray = aPossibleArray => aPossibleArray && Array.isArray(aPossibleArray);
```

Comment:
I think you can remove the eslint disable and export it default like this
`export default aPossibleArray => aPossibleArray && Array.isArray(aPossibleArray);`